### ISSUE TITLE
A11y bug Color Contrast on self generated Thumbnails when shared on Twitter

### ISSUE THREAD

**Issue by YuriDevAT** [forem/forem#18772]
**Describe the bug**

When sharing an article from dev on Twitter, where no thumbnail was created but automatically generated by dev, the text color appears in light gray which is not WCAG2.1 conform. Text to background contrast must be at least 4.5:1.

**To Reproduce**
According to [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) the contrast is only 1.9:1.

Gray #B5BCCA extracted with ColorZilla Browser Extension.

<img width="488" alt="Blog Post with generated Thumbnail by Dev. Text is in light gray and therefore does not meet accessibility requirements of an contrast of at least 4.50:1." src="https://user-images.githubusercontent.com/54622834/203498764-f38ee706-61d2-435f-9e62-11f40797031d.png">

**Screenshots**

<img width="731" alt="Solution of contrast checker with the webaim contrast checker showing that the contrast is 1.9:1." src="https://user-images.githubusercontent.com/54622834/203499405-c9b714a0-4251-4719-8386-b3e5b2bc04e6.png">

**Expected behavior**

Color contrast has to be at least 4.5:1, which would be a minimum of gray #6F749B when increasing hue.

**Screenshots**

**Desktop (please complete the following information):**

- Browser, version: MacBook Air M1, , macOS Monterey 12.6 Firefox 107

**Smartphone (please complete the following information):**

- Device:
- OS, version: iPhone 13 Pro, iOS, 16.1.1

**Additional context**

**Comment by thejwuscript** [comment]
I would like to work on this issue.
